code/datums/loadout/item_representation/armor_representation.dm

```diff
@@ -0,0 +1,154 @@
+/**
+ * Allow to representate a suit that has storage
+ * This is only able to representate items of type /obj/item/clothing/suit/storage
+ * In practice only PAS will use it, but it supports a lot of objects
+ */
+/datum/item_representation/suit_with_storage
+	///The storage of the suit
+	var/datum/item_representation/storage/pockets
+
+/datum/item_representation/suit_with_storage/New(obj/item/item_to_copy)
+	if(!item_to_copy)
+		return
```

Comment:
dont you want this to crash as well

Reply:
No, because of jatum, see discution on disc